### DISCUSSION TITLE
Help please :(

### DISCUSSION THREAD

**Discussion by MrRawrrino89** [DIGITALCRIMINALS/OnlyFans#327] [Q&A]
Anybody know how to fix this part? 

![Screenshot_1](https://user-images.githubusercontent.com/86880907/152081508-351069e1-19f8-4394-a1ed-4c708dc02c52.png)

**Comment by MrRawrrino89** [comment]
now getting this!
![Screenshot_2](https://user-images.githubusercontent.com/86880907/152081674-3188d80c-bb04-409c-a1ba-06bc4ccbe592.png)

**Comment by P34k4k** [comment]
A bit late but this happened to me as well. The way I fixed this error was I deleted all the files then re-downloaded them ( onlyfans- 7.6.1 -zip) and ran the start_ofd (don't run the updater script) 

this is how I have my config set up. hop this helps :
[message.txt](https://github.com/DIGITALCRIMINALS/OnlyFans/files/8335240/message.txt)
